javascript/javascript1/week3/life-time.js

```diff
@@ -10,7 +10,7 @@ const seriesDurations = [
 		title: 'Revolution',
 		days: 1,
 		hours: 5,
-		minutes: 24,
+		minutes: 20,
```

Comment:
I recommend you to covert lifespan from 80 days to minutes as well as series durations like this
```js
  function logOutSeriesText() {
    const lifespan = 80 * 365 * 24 * 60; // minutes
    let total = 0;
    for (let i = 0; i < seriesDurations.length; i++) {
      const seriesDuration = seriesDurations[i].days * 24 * 60 + seriesDurations[i].hours * 60;
      const percent = (seriesDuration / lifespan) * 100;
      total += percent;
      console.log(`${seriesDurations[i].title} took ${percent}% of my life`);
    }
    console.log(`In total that is ${total}% of my life`);
  }
```